### DISCUSSION TITLE
OVA集数错误（NaN）

### DISCUSSION THREAD

**Discussion by BakaJzon** [the1812/Bilibili-Evolved#1548] [功能建议 / Ideas]
这该算啥？特性！
![image](https://user-images.githubusercontent.com/55239052/107730222-d043eb00-6d2d-11eb-9c2f-d19d69606573.png)
![image](https://user-images.githubusercontent.com/55239052/107730237-d934bc80-6d2d-11eb-9a83-235a62b3f07b.png)

**Comment by the1812** [comment]
应该算 bug (

**Comment by FoundTheWOUT** [comment]
emm, 按照我的理解, 当时在我的commit里已经修复了这一类的错误.
https://github.com/FoundTheWOUT/Bilibili-Evolved/commit/a3b7286ec95843a43943df6bd72b8f35e8fc3d9a
而且, 我刚刚找了一圈, 现在OVA之类的都已经分开为另一个合集里面了吧, OVA我没找到在合集里, 但OAD的找到了并测试了一下, 是能正常导出的
![截屏2021-03-26 上午11 25 33](https://user-images.githubusercontent.com/47922588/112572828-fc22b800-8e25-11eb-8fc5-f977c5da41a8.png)
你可以给个链接我看看什么情况吗. 
或者你要更新一下插件?

**Reply by the1812** [reply]
应该修了的, 这个最早是 #1383 里提出的, 在 v1.12.0 里修复
